SimPEG/Utils/modelutils.py

```diff
@@ -1,62 +1,61 @@
 from .matutils import mkvc, ndgrid
 import numpy as np
 
-def surface2ind_topo(mesh, topo, gridLoc='CC'):
-# def genActiveindfromTopo(mesh, topo):
+def surface2ind_topo(mesh, topo, gridLoc='CC', method='cubic',fill_value=np.nan):
     """
     Get active indices from topography
     """
-
+    from scipy.interpolate import griddata
```

Comment:
Imports should be done at the top of the file ideally (inside functions is a performance hit...).